packages/medusa/src/loaders/config.ts

```diff
@@ -72,8 +83,9 @@ export default (rootDirectory: string): ConfigModule => {
 
   return {
     projectConfig: {
-      jwt_secret: jwt_secret ?? "supersecret",
-      cookie_secret: cookie_secret ?? "supersecret",
+      // Question: Should we just introduce the breaking change now, or do we want to stay backward compatible?
```

Comment:
I'd say let's start with a breaking change, and once we have everything working and we are closer to a release we evaluate what we want to be backwards-compatible with and what not.